### ISSUE TITLE
Client Allocation Request for: 广州星淼文化传播有限公司

### ISSUE THREAD

**Issue by denglpo** [filecoin-project/filecoin-plus-client-onboarding#894]
---

# Client Allocation Questions 

## Core Information
- Name: 广州星淼文化传播有限公司
- Website / Social Media: https://cn.hk.uy/Nt6;Wechat:365053027
- Region: Greater China
- DataCap Requested: 10TiB
- Addresses to be Notarized: f3tdrpylphoxvu6sgngnmlmai7my6tapqa3dfw2yipwv5jkns64bm3fvr26rjoust5j5r4hc4gk27dycjf4rya
- Notary Requested: IPFSUnion

@denglpo Please subscribe to notifications for this Issue to be aware of updates. Notaries may request additional information on the Issue.

**Comment by IPFSUnion** [comment]
Thank you for your request. I'd like to know more about your business. Can you kindly answer my following questions?
* individual and organization  
    1.	Please introduce yourself and your team briefly, and show me some your social media accounts such as email address, WeChat official account and other evidence proving that you have the right to apply on behalf of the company.
    2.	For your organization, Please describe its time since inception, registration country，scale, named officers, project, industry reputation etc.
* track record
    1.	Regarding your data you intend to store, What’s the original source of those data, and do you have any proof of ownership or authorization?
    2. Has your organization obtained DataCap before? 
    3. Please list any of your honor and disputes in the field of Filecoin.
* use plan
    1.	Please describe your plan to use DataCap briefly? (such as use case, miner etc.)

**Comment by denglpo** [comment]
Please introduce yourself and your team briefly, and show me some your social media accounts such as email address, WeChat official account and other evidence proving that you have the right to apply on behalf of the company.
For your organization, Please describe its time since inception, registration country，scale, named officers, project, industry reputation etc.
I'm Luo Xue, wechat account:365053027. I'm a entertainment broker. We provide audiovisual and film brokerage agency services. I has rich experience in stage performance and art guidance. I have planned many large activities, literary and art performances, competitions and other public cultural activities. I’m good at maintaining corporate image, planning creativity, conference and exhibition services. We also provide personal image design, stage performance makeup services, photography and art counseling.
Guangzhou Xingmiao Cultural Media Co., Ltd. was founded in 2012, which has a professional executive team, domestic and foreign model team and photography team. Italian fashion planning and design directors and French D&G photographers are invited to join the company, providing high-quality model services for global and domestic shooting, exhibitions, product launches and so on. Professional directors lead the Paris production company to shoot foreign films, travel, etc. Cooperate with professional external mold team to meet various shooting needs.
Regarding your data you intend to store, What’s the original source of those data, and do you have any proof of ownership or authorization?
The data we intend to store on the Filecoin network is publicly available, mainly including video materials, advertising. The videos come from various large stage performances, such as parties, sports events, festivals, art modeling contests, art festivals, public benefit performances and exhibitions, etc. The format is document, video, and audio. The total is about 50T.
Has your organization obtained DataCap before?
None.
Please list any of your honor and disputes in the field of Filecoin.
None.
Please describe your plan to use DataCap briefly? (such as use case, miner etc.)
We plan to select about three miners.

**Comment by IPFSUnion** [comment]
About the website/Social Media you provide
  1. The company described in https://cn.hk.uy/Nt6 is same as https://github.com/filecoin-project/filecoin-plus-client-onboarding/issues/906
  2. I could not get in touch with the WeChat account you provided, and the other party didn't have knowlege of this application.
  Based on the above doubts, I can't trust and allocate datecap to you this time.

**Comment by denglpo** [comment]
We are in the same industry with 广州尚恩文化传播有限公司, and we recommended Filecoin to them after we learned that filecoin can be stored at a low price.     When they showed us their website information, we mistakenly posted their information.     We have corrected it.    **Because** we are a traditional industry and don't do website, but we do have a lot of data. In addition, Luo is busy on a business trip, so you have not contacted her.  We can submit our business license and sample data to prove the authenticity of our business.
